test/startservers.py

```diff
@@ -64,11 +64,11 @@
         ('sd-test-srv', 'boulder-remoteva-a', 'boulder-remoteva-b')),
     Service('boulder-ca-a',
         8001, 'ca1.boulder:9093',
-        ('./bin/boulder-ca', '--config', os.path.join(config_dir, 'ca-a.json'), '--ca-addr', 'ca1.boulder:9093', '--ocsp-addr', 'ca1.boulder:9096', '--crl-addr', 'ca1.boulder:9196', '--debug-addr', ':8001'),
+        ('./bin/boulder-ca', '--config', os.path.join(config_dir, 'ca-a.json'), '--ca-addr', 'ca1.boulder:9093', '--ocsp-addr', 'ca1.boulder:9096', '--crl-addr', 'ca1.boulder:9106', '--debug-addr', ':8001'),
```

Comment:
Because it was set to the wrong thing by the previous PR, but the error wasn't detected until there was a running crl-updater to try to actually talk to it. (Compare with the address set in the actual ca.json: https://github.com/letsencrypt/boulder/blob/main/test/config-next/ca-a.json#L29)